api/spec/controllers/tournaments_controller_spec.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe TournamentsController, type: :controller do
+  describe 'GET #index' do
+    it 'returns http 401 when missing token' do
+      get :index, format: :json
+      expect(response).to have_http_status(401)
+    end
+
+    it 'returns http 401 when expired token' do
+      exired_http_login
+      get :index, format: :json
+      expect(response).to have_http_status(401)
+    end
+
+    it 'returns empty array' do
+      http_login
+      get :index, format: :json
+      expect(response).to have_http_status(:success)
+      expect(response.body).to be_json_eql([])
+    end
+
+    it 'returns tournaments' do
+      tournaments = create_list(:tournament, 5)
+
+      http_login
+      get :index, format: :json
+      expect(response).to have_http_status(:success)
+      expect(response.body).to be_json_eql(tournaments.map { |t| { id: t.id, name: t.name } }.to_json)
```

Comment:
Please do not test like this. Add Torunaments with name manually and check json manually